include/cantera/zerodim.h

```diff
@@ -1,13 +1,29 @@
+//! @file zerodim.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
 #ifndef CT_INCL_ZERODIM_H
 #define CT_INCL_ZERODIM_H
-#include "zeroD/Reactor.h"
+
+// reactor network
 #include "zeroD/ReactorNet.h"
-#include "zeroD/Reservoir.h"
-#include "zeroD/Wall.h"
-#include "zeroD/flowControllers.h"
-#include "zeroD/FlowReactor.h"
-#include "zeroD/ConstPressureReactor.h"
-#include "zeroD/IdealGasReactor.h"
-#include "zeroD/IdealGasConstPressureReactor.h"
+
+// reactors
+#include "cantera/zeroD/Reservoir.h"
+#include "cantera/zeroD/Reactor.h"
+#include "cantera/zeroD/FlowReactor.h"
+#include "cantera/zeroD/ConstPressureReactor.h"
+#include "cantera/zeroD/IdealGasReactor.h"
+#include "cantera/zeroD/IdealGasConstPressureReactor.h"
+
+// flow devices
+#include "cantera/zeroD/flowControllers.h"
+
+// walls
+#include "cantera/zeroD/Wall.h"
+
+// surface
+#include "cantera/zeroD/ReactorSurface.h"
```

Comment:
I think the factory class header files should be included here as well, which would be consistent with `thermo.h` and `kinetics.h`, as well as saving you a few other `#include`s elsewhere.

Reply:
done.